searchcommon/src/vespa/searchcommon/attribute/i_attribute_functor.h

```diff
@@ -14,8 +16,14 @@ namespace search::attribute {
 class IAttributeFunctor
 {
 public:
-    virtual void operator()(const search::AttributeVector &attributeVector) = 0;
+    virtual void operator()(const IAttributeVector &attributeVector) = 0;
```

Comment:
maybe the attribute vector should be non-const, since we intend to modify it?

Reply:
This was an interface that existed earlier on. I did not change it.
I think I agree.